akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -593,7 +593,7 @@ final class GraphInterpreter(
       while (eventsRemaining > 0 && queueTail != queueHead) {
         val connection = dequeue()
         eventsRemaining -= 1
-        chaseCounter = math.min(ChaseLimit, eventsRemaining)
+        chaseCounter = if (fuzzingMode) 0 else math.min(ChaseLimit, eventsRemaining)
```

Comment:
I kept it as a compile-time constant, although this might be a bit overdoing it. Still, I gained performance on *very* tiny things so far, every few nanosecond seem to count now.